resources/fixtures/American_DJ/American-DJ-Entour-Faze.qxf

```diff
@@ -26,6 +26,7 @@
  <Channel Name="Interval">
   <Group Byte="0">Speed</Group>
   <Capability Min="0" Max="1">0 Seconds</Capability>
+  <Capability Min="2" Max="8">No function</Capability>
```

Comment:
Your values are for duration, this is the interval capability with
0-1: 0s
9-255: 5-255s

In this case, there is a gap in the manual for 2-8, which effectively is a "reserved / no function" in my perception and could be the same as "no interval (0s)". It could as well mean "2-9: 1-4s" and be forgotten in the manual. The real behavior can probably only be resolved with the real device at hand.
Thus the proposal to fill this with a "No Function" as long as no-one complains.

Reply:
Ok